src/promptflow/promptflow/_cli/_pf/_init_entry_generators.py

```diff
@@ -401,14 +402,15 @@ def entry_template_keys(self):
 class ToolPackageUtilsGenerator(BaseGenerator):
     def __init__(self, package_name):
         self.package_name = package_name
+        self.package_uuid = str(uuid4())
```

Comment:
why is this an randome guid?

Reply:
Because there is no guarantee that users will have the same name package in different Python environments